app/models/sourcesmodel.py

```diff
@@ -1,30 +1,28 @@
-from app.models.basemodel import Base
+import json
+from externalapi import sources_feed
 from sqlalchemy.orm import relationship, backref
-from app.models.allnewsmodel import AllNewsModel
-from app.models.headlinesmodel import HeadlineModel
 from app.config import db
 
-class SourceModel(Base):
-    __tablename__ ='sources'
-    __table_args__ = {'sqlite_autoincrement': True}
 
+class SourceModel(db.Model):
+    __tablename__ ='sources'
+    __table_args__ = {'extend_existing': True}
 
-    headlines_id =db.Column(db.Integer,db.ForeignKey('headlines.id'))
-    allnews_id = db.Column(db.Integer, db.ForeignKey('allnews.id'))
+    id = db.Column(db.Integer, primary_key=True, unique=True, nullable=False, autoincrement=True)
+   # headlines_id =db.Column(db.Integer, db.ForeignKey('headlines.headline_id'))
+    #allnews_id = db.Column(db.Integer, db.ForeignKey('allnews.allnews_id'))
```

Comment:
If these lines are meant to be removed please remove them altogether, don't leave them commented in your final version.